mindmeld/tokenizer.py

```diff
@@ -134,28 +158,6 @@ def __deepcopy__(self, memo):
         # TODO: optimize this
         return Tokenizer(exclude_from_norm=self.exclude_from_norm)
```

Comment:
This isn't going to produce a faithful copy anymore.

Reply:
Good point. As you mentioned this is because the app_path is not being passed in upon instantiation. To resolve this, I've made two changes. Firstly, the app_path is now being saved as a class attribute and secondly, the app_path is being passed on instantiation.

I tested this out in a notebook and this is now creating the correct internal tokenizer and normalizer objects

`return Tokenizer(app_path=self.app_path, exclude_from_norm=self.exclude_from_norm)`